get-started/codelab/index.md

```diff
@@ -25,11 +31,11 @@ previous experience with Dart or mobile programming.
 
 You’ll implement a simple mobile app that generates proposed names for a
 startup company. The user can select and unselect names,
-saving the best ones. The code lazily generates ten names at a time.
-As the user scrolls, new batches of names are generated.
-The user can scroll forever, with new names being continually generated.
+saving the best ones. The code lazily generates names.
+As the user scrolls, more names are generated.
+There is no limit to how far a user can scroll.
 
-The animated GIF shows how the app works at the completion of part 1:
+The animated GIF shows how the app works at the completion of Part 1.
```

Comment:
I suppose that this should be "part 1" (lowercase) like the rest of the occurrences of the word?